index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/0278cc9d4c.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@200;300;400;500;700&display=swap" rel="stylesheet">
+
+    <link rel="stylesheet" href="style.css">
+    <title>John Doe</title>
+</head>
+<body>
+    <nav class="navbar">
+        <div class="navbar__logo">
+            John Doe
+        </div>
+        <div class="navbar__menu">
+            <i class="fa-solid fa-bars"></i>
+        </div>
+    </nav>
+    <header class="header">
+        <h1 class="heading__primary">Hey There.
+            I’m Alex
+        </h1>
+        <h2 class="heading__secondary">
+            I’m a Software Developer
+        </h2>
+        <p class="heading__about">
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+don’t hestiate to contact me.
+        </p>
+        <div class="header__social">
+            <div class="header__icon">
+                <i class="fa-brands fa-github"></i>
+            </div>
+            <div class="header__icon">
+                <i class="fa-brands fa-linkedin-in"></i>
+            </div>
+            <div class="header__icon">
+                <i class="fa-brands fa-angellist"></i>
+            </div>
+            <div class="header__icon">
+                <i class="fa-brands fa-twitter"></i>
+            </div>
+            <div class="header__icon">
+                <i class="fa-brands fa-medium-m"></i>
+            </div>
+            
+        </div>
```

Comment:
- [ ] Kindly use `ul` instead of `div`, and `li` and `a` for each icon.

like
```
<ul>
  <li><a href=""><img ..../><a/></li>
  ............
</ul>
```